speaker_verification/tests/test_sqlmethods.py

```diff
@@ -0,0 +1,95 @@
+import unittest
+
+import pytest
+import numpy as np
+from os.path import join, abspath, dirname
+import pathlib
+import sqlite3
+import unittest
+
+import pytest
+import numpy as np
+from os.path import join, abspath, dirname
+import pathlib
+import sqlite3
+
+from speaker_verification.sql_utils import(
+    create_db_table,
+    insert_db_row,
+    select_db_row,
+    remove_db_row
+)
+

```

Comment:
I think we could set this up in a `pytest.fixture` instead, this might be fine if its too hard to create the test db in memory